corehq/apps/app_manager/models.py

```diff
@@ -3012,6 +3012,12 @@ def actions(self):
             shadow_parent_actions = self.shadow_parent_form.actions
         return self._merge_actions(shadow_parent_actions, self.extra_actions)
 
+    def get_shadow_parent_options(self):
```

Comment:
Would be nice for this to also handle the case where the form has a shadow parent but it isn't in `app.get_forms` giving it a name like `Unknown, please change`. Someone's going to set up a shadow form, delete the parent form accidentally, and then have trouble fixing the build error.